src/components/Checkout/DonateRepaymentsToggle.vue

```diff
@@ -1,29 +1,52 @@
 <template>
-	<div v-if="showToggle" class="donate-repayments-toggle">
-		<kv-checkbox
-			class="donate-repayments-label"
-			id="donate-repayments"
-			:checkbox-right="true"
-			v-if="!myDonateRepayments"
-			v-model="donateRepayments"
-			@change="toggleDonateRepayments"
+	<div>
+		<div
+			v-if="showToggle"
+			class="tw-mt-2 tw-flex donate-repayments-toggle"
 		>
-			<span
-				id="donate-repayments-tooltip"
-				class="tw-text-small tw-text-link tw-font-medium"
+			<button
+				class="tw-flex-1 tw-pt-0.5 md:tw-pt-0 tw-text-base
+					tw-font-medium tw-text-left md:tw-text-right tw-mr-2
+					tw-cursor-pointer"
+				@click="toggleCheckbox"
+				aria-hidden="true"
 			>
 				Donate loan repayments instead?
-			</span>
-		</kv-checkbox>
-		<kv-tooltip controller="donate-repayments-tooltip" theme="mint">
-			<template #title>
-				Thanks for your support!
-			</template>
-			When you check this box, repayments go back to Kiva in the form of donations,
-			helping us cover operating costs and reach even more borrowers worldwide.
-			<br>
-			Repayments from selected loans will not be added back to your account as Kiva credit.
-		</kv-tooltip>
+			</button>
+			<kv-checkbox
+				class="tw-relative tw-cursor-pointer md:tw-text-right kv-checkbox tw-pt-0 donate-repayments-label"
+				id="donate-repayments"
+				data-testid="donate-repayments"
+				v-if="!myDonateRepayments"
+				v-model="donateRepayments"
+				@change="toggleDonateRepayments"
+			>
+				<span
+					id="donate-repayments-tooltip"
+					data-testid="donate-repayments-tooltip"
+					class="tw-text-small tw-font-medium tw-sr-only"
+				>
+					Donate loan repayments instead?
+				</span>
+			</kv-checkbox>
+			<kv-lightbox
+				:visible="isLightboxVisible"
+				title="Donate your loan repayments"
+				@lightbox-closed="closeLightbox"
+			>
+				When you check this box, repayments go back to Kiva in the form of donations,
+				helping us cover operating costs and reach even more borrowers worldwide.
+				Repayments from selected loans will not be added back to your account as Kiva credit.
+			</kv-lightbox>
+		</div>
+		<div class="tw-flex tw-mt-2">
+			<kv-text-link
+				@click="isLightboxVisible = true;"
+				class="tw-text-base tw-text-left md:tw-text-right tw-w-full"
```

Comment:
Not sure if the `tw-text-base` will always take effect, but I haven't tested it. I think kv-text-link wants to be `tw-text-h4` generally.